addons/web/static/src/legacy/js/public/public_root.js

```diff
@@ -6,8 +6,7 @@ import env from 'web.public_env';
 import session from 'web.session';
 import utils from 'web.utils';
 import publicWidget from 'web.public.widget';
-
-var publicRootRegistry = new publicWidget.RootWidgetRegistry();
+import { registry } from '../../../core/registry';
```

Comment:
The idea is generally to try to use relative imports when in the same module, because that allows editor tooling like VSCode to find the file and have auto-completion without having to define resolve paths explicitly in your tsconfig/jsconfig, but this convention is followed quite loosely, and in this case I agree that the path makes more sense when reading if it's an absolute path so I'll change it.